source/download/index.html.haml

```diff
@@ -35,7 +49,7 @@ The <a href="https://fedoraproject.org/wiki/Cloud_SIG">Fedora Cloud SIG</a> offe
   %a.btn.btn-primary.btn-lg{:href => "https://wiki.centos.org/SpecialInterestGroup/Atomic/Download/"} CentOS Atomic Host
 
 %p.buttons.hidden-xs
-  %a.btn.btn-primary.btn-lg{:href => "https://getfedora.org/atomic/download/"} Fedora Atomic Host
+  %a.btn.btn-primary.btn-lg{:href => "https://dl.fedoraproject.org/pub/alt/atomic/stable/"} Fedora Atomic Host
```

Comment:
There's also `https://getfedora.org/atomic_qcow2_latest`, which will redirect to the latest qcow2.

Reply:
i thought that had stopped working but looks like it is working again

Reply:
I'm going to leave the directory, in the off chance someone wants a non-x86_64 version of the qcow or other artifact.